R/plotQQnorm.R

```diff
@@ -35,6 +38,10 @@ plotQQnorm <- function(residuals, lower = NULL, upper = NULL, abline = TRUE, abl
     df$ymax <- upper
   }
 
+  if (isTRUE(na.rm)) {
+    df <- df[isFALSE(is.na(residuals)), ]
+  }
```

Comment:
`isFALSE` is defined as 
```r
is.logical(x) && length(x) == 1L && !is.na(x) && !x
```
so it checks `length(x) == 1L`. I don't think that is what you want?

any reason not to just do this?
```suggestion
  if (isTRUE(na.rm)) {
    df <- df[is.na(residuals), ]
  }
```

Reply:
Ah yes did not know that, I thought it would just flip the logical, like "!" and I thought that was the preferred way. I'll. change it to (your code suggestion would only keep the NA's):
```
  if (isTRUE(na.rm)) {
    df <- df[!(is.na(residuals)), ]
  }
```